### ISSUE TITLE
per child/pane startup commands are excessively difficult to configure

### ISSUE THREAD

**Issue by justin-barca** [gnunn1/tilix#1527]
I spent a lot of time browsing all of the issues in this repo and I could not find a solution to running per-pane commands in a saved configuration.  There was some talk of using `bash -c` but none of the solutions really worked properly for my application.  I am posting this more for the purposes of documentation than to ask for a feature.  

If you need to run commands after opening a shell, here's an example of a custom command that will enable this:
`/bin/zsh -c 'source ~/.zshrc && cd ~/dev/storefront-next && npm run server; exec /bin/zsh'`
(of course, if you're using bash you will have to modify this to point to the bash command and bashrc file instead).

A nice to have would be a config property that would allow me to specify just an init command like `cd ~/dev/storefront-next && npm run server` in both profiles and layout

**Comment by gnunn1** [comment]
Tilix already has this ability in both profiles and layout. For the profile, it is under the command tab, "Run a custom command instead of my shell". For layouts, in the terminal titlebar select Other > Layout Options and specify the command under Session Load.

I agree this could be more intuitive, if someone wants to suggest a better scheme  or do a mockup that would be much appreciated.

![screenshot from 2018-09-18 17-08-08](https://user-images.githubusercontent.com/9358238/45716678-99217d80-bb65-11e8-8c1d-36c5ee9d75c0.png)

**Comment by justin-barca** [comment]
Yes, btw, I really do like how configurable this is, you have done a great
job! just another input below the custom command called 'init command' to
mirror the verbiage of the bash command line init file arg would be great.
